uint8 ADD=0
uint8 DELETE=2
uint8 DELETEALL=3

int32  action
string name
string desc
